config/software/libsodium.rb

```diff
@@ -17,35 +17,16 @@
 
 # We use the version in util-linux, and only build the libuuid subdirectory
 name "libsodium"
-default_version "1.0.17"
+default_version "1.0.18"
 
 license "ISC"
 license_file "LICENSE"
 skip_transitive_dependency_licensing true
 
-version "1.0.17" do
-  source sha256: "0cc3dae33e642cc187b5ceb467e0ad0e1b51dcba577de1190e9ffa17766ac2b1"
-end
+version("1.0.17") { source sha256: "0cc3dae33e642cc187b5ceb467e0ad0e1b51dcba577de1190e9ffa17766ac2b1" }
+version("1.0.18") { source sha256: "0cc3dae33e642cc187b5ceb467e0ad0e1b51dcba577de1190e9ffa17766ac2b1" }
```

Comment:
That would be quite a coincidence for two subsequent software releases to have the same shasum.

Reply:
Fixed